doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,138 @@
+# Environment Variables in Julia
+
+The environment variables that Julia uses generally start with `JULIA`. If [`Base.versioninfo()`](https://github.com/JuliaLang/julia/blob/master/base/interactiveutil.jl#L247) is called with `verbose` equal to `true`, then the output will list all environment variables for which `JULIA` appears in the name.
+
+## File locations
+
+### `JULIA_HOME`
+The absolute path of the directory containing the Julia executable, which sets the global variable [`Base.JULIA_HOME`](https://github.com/JuliaLang/julia/blob/master/base/initdefs.jl#L90).  If `$JULIA_HOME` is not set, then `Base.JULIA_HOME` defaults to its compile-time value.
```

Comment:
It's a little mixed in the existing markdown files, but I think in general we should be using a single space after periods (or a line break—that also works) rather than two spaces.